docs/framework/install/on-windows-8.md

```diff
@@ -13,22 +13,35 @@ ms.devlang: dotnet
 
 # Install the .NET Framework on Windows 8
 
-You may need the .NET Framework to run an app on Windows 8. You can use the following instructions to help you. The instructions depend on which version of the .NET Framework you need.
+The .NET Framework is required to run many applications on Windows. You can use the following instructions to install it. You may have arrived on this page after trying to run an application and seeing the following dialog on your machine.
 
-## .NET Framework 4.7.1
+![This application could not be started](./media/this-application-could-not-be-started.png)
 
-You can install the [.NET Framework 4.7.1](https://www.microsoft.com/net/download/framework) on Windows 8.
+These instructions will help you install the .NET Framework versions you need. The .NET Framework 4.7.1 is the latest version, however, it is not supported on Windows 8.
```

Comment:
version, however, --> version. However,
Perhaps mention here that if you want a later version, you must upgrade?